automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -459,11 +490,13 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
               implicit val authToken: AuthToken = authTokenDefault
               withGroup("AuthDomain", List(user.email)) { groupOneName =>
                 withGroup("AuthDomain", List(user.email)) { groupTwoName =>
-                  withWorkspace(projectName, "AuthDomainSpec_share", Set(groupOneName, groupTwoName)) { workspaceName =>
-                    withSignIn(user) { listPage =>
-                      val summaryPage = listPage.enterWorkspace(projectName, workspaceName)
-                      summaryPage.readAuthDomainGroups should include(groupOneName)
-                      summaryPage.readAuthDomainGroups should include(groupTwoName)
+                  withCleanBillingProject(defaultUser) { projectName =>
+                    withWorkspace(projectName, "AuthDomainSpec_share", Set(groupOneName, groupTwoName)) { workspaceName =>
```

Comment:
This workspace needs to be created by the local `user` in order to test the billing project owner's special access (as long as they're in all of the groups making up the auth domain).

Reply:
Updated tests with your corrections.  Thanks!